src/makeRobot.js

```diff
@@ -39,6 +54,94 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  const robot = {
+    'name': name,
+    'chip version': version,
+    'wheels': wheels,
+    'coords': {
```

Comment:
u don't need to explicitly make them strings, all object keys are string by default
```suggestion
    coords: {
```